### PR TITLE
WebDriver: Implement `POST /session/{id}/refresh` endpoint

### PR DESCRIPTION
**Pull request by TobyAsE**
Adds a tick to issue #15551 @AtkinsSJ

### CODE REVIEW

File: Userland/Services/WebDriver/Session.cpp [hunk 1]
```diff
@@ -158,4 +158,30 @@ ErrorOr<JsonValue, HttpError> Session::get_title()
     return JsonValue(m_browser_connection->get_title());
 }
 
+// POST /session/{session id}/refresh https://w3c.github.io/webdriver/#dfn-refresh
+ErrorOr<void> Session::refresh()
+{
+
```
Comment (AtkinsSJ) [inline Userland/Services/WebDriver/Session.cpp:164]: The nittest of picks: Weird empty line here.

**Review by AtkinsSJ — CHANGES_REQUESTED** [review]
LGTM, just one small thing

File: Userland/Services/WebDriver/Session.cpp (hunk 1 above)
Comment (TobyAsE) [reply Userland/Services/WebDriver/Session.cpp]: more love to nitpicks! :^)

**Review by AtkinsSJ — APPROVED** [review]

File: Userland/Services/WebDriver/Session.cpp [hunk 2]
```diff
@@ -158,4 +158,29 @@ ErrorOr<JsonValue, HttpError> Session::get_title()
     return JsonValue(m_browser_connection->get_title());
 }
 
+// POST /session/{session id}/refresh https://w3c.github.io/webdriver/#dfn-refresh
+ErrorOr<void> Session::refresh()
+{
+    // 1. If the current top-level browsing context is no longer open, return error with error code no such window.
+    auto current_window = get_window_object();
+    if (!current_window.has_value())
+        return HttpError { 400, "no such window", "Window not found" };
```
Comment (linusg) [inline Userland/Services/WebDriver/Session.cpp:167]: 404

**Review by linusg — CHANGES_REQUESTED** [review]

File: Userland/Services/WebDriver/Session.cpp (hunk 2 above)
Comment (linusg) [reply Userland/Services/WebDriver/Session.cpp:167]: https://w3c.github.io/webdriver/#dfn-no-such-window

Comment (TobyAsE) [reply Userland/Services/WebDriver/Session.cpp:167]: than its wrong for the others, too. will fix.

**Review by linusg — APPROVED** [review]

**Review by linusg — COMMENTED** [review]
The CI build is not happy though :thinking:

**Pull request merged**